x-pack/test/security_solution_cypress/cypress/tasks/alerts.ts

```diff
@@ -198,23 +196,16 @@ export const closePageFilterPopover = (filterIndex: number) => {
 };
 
 export const clearAllSelections = (filterIndex: number) => {
-  cy.get(CONTROL_GROUP).scrollIntoView();
-  recurse(
-    () => {
-      cy.get(CONTROL_FRAME_TITLE).eq(filterIndex).realHover();
-      return cy.get(OPTION_LIST_CLEAR_BTN).eq(filterIndex);
-    },
-    ($el) => $el.is(':visible')
-  );
-  cy.get(OPTION_LIST_CLEAR_BTN).eq(filterIndex).should('be.visible').trigger('click');
+  cy.get(OPTION_LIST_VALUES(filterIndex)).realHover();
```

Comment:
One more thing. Hopefully it doesn't happen, but I've seen `realHover` be flaky in the past

Reply:
You know @MadameSheema had pointed out to me but from last 2 days i have been trying to make `mouseover` and `mouseenter` but they `don't` work consistently and `realHover` works 100% of the time.

Reply:
In case you did not see Flaky Test Runner results : https://github.com/elastic/kibana/pull/173163#issuecomment-2071840263